styles/styles.css

```diff
@@ -0,0 +1,130 @@
+/* color variables */
+:root {
+  --overlay-clr: rgb(234, 234, 236, 0.6);
+}
+
+/* navigation */
+.navigation-container {
+  margin: 0;
+  width: inherit;
+  background-color: var(--white-variant);
+}
+
+/* home page display category */
+
+.home-display-category-container {
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  gap: 0.5rem;
+  margin: 2rem;
+}
+
+.home-display-category {
+  max-width: 300px;
+  color: var(--primary);
+  position: relative;
+  justify-self: center;
+}
+
+.home-display-category:hover {
+  box-shadow: rgba(0, 0, 0, 0.2) 0px 20px 30px;
```

Comment:
You might use this shadow again so better store it as a variable.

Reply:
Thank you for the idea, I'll do that for sure